get-config-state.js

```diff
@@ -40,6 +40,10 @@ function getConfigState(dirname, opts) {
         dc && NODE_ENV ?
             join(configFolder, NODE_ENV + '.' + dc.datacenter + '.json') :
             null,
+        // load ./config/secrets-NODE_ENV.json
+        NODE_ENV ? join(configFolder, 'secrets-' + NODE_ENV + '.json') : null,
+        // load ./config/secrets.json
```

Comment:
I'm trying to get clarity on if our standard practice will be `config/secrets/secrets.json` or `config/secrets.json` I picked the latter for the PR to bikeshed on.